### PR TITLE
Bug fix for MPSoC interrupt when OpenCL API is using

### PR DESCRIPTION
**Pull request by mamin506**
OpenCL layer is not going to enable interrupt in regmap. MicroBlaze ERT will enable CU interrupt based on ert_polling configure. For MPSoC platform, KDS need to enable interrupt, otherwise the user's application will hang on interrupt mode.

XILINX_XRT should point to /usr for MPSoC platform. Since all of the header files and libraries are installed in linux standard directory.

Other refine:
Skip 0x4, 0x8, 0xC register configure when start CU. They are used for enable interrupt and identify CU interrupt status. Apply for both xocl and zocl.

### CODE REVIEW

**Comment by maxzhen** [comment]
ok to test

**Comment by NaAl-Xlnx** [comment]
retest this please

**Comment by mamin506** [comment]
retest this please

File: src/runtime_src/driver/zynq/drm/zocl/sched_exec.c [hunk 1]
```diff
@@ -537,10 +559,15 @@ configure(struct sched_cmd *cmd)
 		    "request %d CUs. Fall back to polling mode\n",
```
[the remark this answers is not in the document]
Comment (mamin506) [reply src/runtime_src/driver/zynq/drm/zocl/sched_exec.c]: done. Thanks.

**Comment by mamin506** [comment]
retest this please

**Pull request merged**